config/settings.yml

```diff
@@ -551,7 +551,6 @@ mhv:
     mock: true
     host: https://apigw-intb.aws.myhealth.va.gov
     access_key: some-access-key
-    create_after_login: false
```

Comment:
@rileyanderson I see this defined in the manifest repo still: https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/apps/vets-api/prod/values.yaml#L2154. Can you delete from there as well?